src/bin/nydus-image/core/context.rs

```diff
@@ -51,6 +52,17 @@ impl Default for RafsVersion {
     }
 }
 
+impl From<u32> for RafsVersion {
```

Comment:
Should we implement TryFrom instead of From so it could return errors.

Reply:
fixed